index.js

```diff
@@ -38,7 +38,7 @@ function handleArgs(cmd, args, opts) {
 	}, parsed.options);
```

Comment:
Can you define the defaults for `cwd` and `localDir` in the Object.assign call instead? Better to have a unified place for defaults.